### PR TITLE
4th milestone - Added logic.

### PR DESCRIPTION
**Pull request by sergiomauz**
This PR contains the logic of the calculator in calculate.js and operate.js files

The link for the Heroku App is [here](https://sz-calculator-reactjs.herokuapp.com/)

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import Big from 'big.js';
+import operate from './operate';
```
Comment (jofer86) [inline src/logic/calculate.js:2]: - We should only be importing operate, big is not part of this module, and should not be included. Try to move the logic you have using Big library to the operate module. As you can see the use of big library is not part of the requirements for this module 
image

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
```
Comment (jofer86) [inline src/logic/operate.js:1]: - Here is the only place where big should be present, remember you are wrapping your values here with the big library so all of the logic using big.js should be placed here.

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- Changes marked as **[OPTIONAL]** are not hard requirements but are nice to have.
- Please see the comments below to check on the **REQUIRED** changes.
- It's a simple observation but it has to be done that way for this milestone to be approved.
## ` const STATUS = 'Changes Required';` 😿 😿 😿
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Review by suretrust — APPROVED** [review]
Hi @sergiomauz, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future milestones and happy coding!_ 🎉

**Pull request merged**